layouts/_default/_markup/render-image.html

```diff
@@ -24,7 +25,7 @@
   {{ with $resource }}
     <figure>
       <picture>
-        {{ if ne .MediaType.SubType "svg" }}
+        {{ if (and (ne .MediaType.SubType "svg") $webp) }}
```

Comment:
Would this break svg support?  
I'm probably wrong, but in my mind, This evaluation is 
if (  (not svg) AND $webp (which is a param bool)  )
which I THINK would undesirably evaluate false in the case of:
is_svg? -> true 
webp = false

I could be wrong tho.... I'm wrong a lot.

Reply:
it will evaluate to false. But this branch is not meant for SVG. SVG is displayed in other branches

Reply:
Since I wrote this code I come with more advanced and cleaner solution https://github.com/stereobooster/hugo-ideal-image/blob/main/layouts/partials/picture.html. I asked author if they want it to be integrated